kili/queries/project/__init__.py

```diff
@@ -63,9 +63,6 @@ def projects(self,
             skip: Number of projects to skip (they are ordered by their creation).
             fields: All the fields to request among the possible fields for the projects.
                 See [the documentation](https://cloud.kili-technology.com/docs/python-graphql-api/graphql-api/#project) for all possible fields.
-                Notice that the field "titleAndDescription" is deprecated since: 18/05/2022.
```

Comment:
I think you also need to remove line 30 `@deprecate(removed_in="2.113")`can you confirm @theodu ?

Reply:
yes Indeed!

Reply:
Line removed